operational_analysis/methods/plant_analysis.py

```diff
@@ -28,6 +28,36 @@
 
 logger = logging.getLogger(__name__)
 
+
+def get_annual_values(data):
+    """
+    This function returns annual summations of values in a pandas Series (or each column of a pandas DataFrame) with a
+    DatetimeIndex index starting from the first row. The purpose of the function is to correctly resample to annual
+    values when the first index does not fall on the beginning of the month.
+
+    Args:
+        data(:obj:`pandas.Series or pandas.DataFrame`): Input data with a DatetimeIndex index.
```

Comment:
@RHammond2 Do you know how to indicate that multiple inputs are allowed? Not sure if this is right.